README.md

```diff
@@ -14,7 +14,51 @@ and run `psp build` to be interactively prompted for configuration values.
 This will generate a `config.json` file that may be used to speed-up future
 invocations.
 
-## Building and deploying policies
+## TL;DR
+
+Run the following command to install the policy builder and build the policies.
+You will be prompted for a few inputs, such as company name, to be included in
+your policy text.
+
+```bash
+npm install -g @jupiterone/jupiter-policy-builder
+
+psp build
+```
+
+You will be prompted to save the config to a file, which you can reference the
+next time you'd like to rebuild the policies and procedures:
+
+```bash
+psp build -t ./templates -c path/to/your/config.json
+```
+
+The result files are put in `./docs` (Markdown) and `./site` (HTML).
+
+**IMPORTANT:** To edit the policies and procedures, use the template files in
+`./templates` and re-run the `psp build` command. Do _not_ edit the `./docs` and
+`./partials` files directly as they will be overwritten on the next build.
+
+For more detailed builder instructions, see the README [here][builder].
```

Comment:
I prefer `see the [README][builder]` over "here" links, but this may be [an old-school position](https://uxmovement.com/content/why-your-links-should-never-say-click-here/).